### DISCUSSION TITLE
gleam remove DEP works on nonexisting deps

### DISCUSSION THREAD

**Discussion by inoas** [gleam-lang/gleam#2964] [Ideas & suggestions]
- could exit with a 0 and print a warning if the dep is not found.
- could get a flag such as --ignore-non-existing to keep current behavior

**Comment by lpil** [comment]
What problem would this solve?

**Comment by inoas** [comment]
1. so if you write gleam remove foo it should by default show an error if no such dep existed.
2. but if you want to just remove something in a script if exists it should also be possible: `gleam remove --if-exists foo` without a non 0 exit code.
